### PR TITLE
Bring `gigasecond` up to date with canonical data

### PR DESCRIPTION
**Pull request by angelikatyborska**

### CODE REVIEW

File: exercises/practice/gigasecond/test/gigasecond_test.exs [hunk 1]
```diff
@@ -2,20 +2,30 @@ defmodule GigasecondTest do
   use ExUnit.Case
 
   # @tag :pending
-  test "from 4/25/2011" do
+  test "from 2011-04-25 00:00:00" do
```
Comment (angelikatyborska) [inline exercises/practice/gigasecond/test/gigasecond_test.exs:5]: More international and programmer-friendly date format...

File: exercises/practice/gigasecond/lib/gigasecond.ex [hunk 2]
```diff
@@ -3,8 +3,7 @@ defmodule Gigasecond do
   Calculate a date one billion seconds after an input date.
   """
   @spec from({{pos_integer, pos_integer, pos_integer}, {pos_integer, pos_integer, pos_integer}}) ::
-          :calendar.datetime()
-
+          {{pos_integer, pos_integer, pos_integer}, {pos_integer, pos_integer, pos_integer}}
```
Comment (angelikatyborska) [inline exercises/practice/gigasecond/lib/gigasecond.ex:6]: The input and output are of the same type, so they should be the same in the type spec. I think `{{pos_integer, pos_integer, pos_integer}, {pos_integer, pos_integer, pos_integer}}` is way uglier to write, but more clear to beginners. I'm not sure...

Comment (neenjaw) [reply exercises/practice/gigasecond/lib/gigasecond.ex:6]: 🤷🏻 either or, I'm not sure that detail is limiting.

File: exercises/practice/gigasecond/test/gigasecond_test.exs (hunk 1 above)
Comment (neenjaw) [reply exercises/practice/gigasecond/test/gigasecond_test.exs:5]: nice usability change

**Review by neenjaw — APPROVED** [review]

**Pull request merged**